ui/widgets/nuxeo-collapsible.js

```diff
@@ -0,0 +1,174 @@
+import '@polymer/iron-icons/hardware-icons.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+{
+  /**
+   * An element for creating a collapsible block of content.
+   * It comes with a default chevron that can be displayed at the end (default) or at the beggining of the heading.
+   * The heading can be expanded by default.
+   * The chevron has its own slot so it can be also customized.
+   * It is also possible to disable the default animation.
+   *
+   * Example:
+   *
+   *     <nuxeo-collapsible chevron chevronAtStart expanded>
+   *        <label slot="heading">Nuxeo</label>
+   *        <div>Collapsed content</div>
+   *     </nuxeo-collapsible>
+   *
+   * ### Styling
+   * The following custom properties and mixins are available for styling:
+   * Custom property | Description | Default
+   * ----------------|-------------|----------
+   * `--nuxeo-collapsible-heading-padding` | Heading padding | `4px`
+   * `--nuxeo-collapsible-icon-height` | Chevron height | `20px`
+   * `--nuxeo-collapsible-icon-width` | Chevron width | `20px`
+   *
+   * @memberof Nuxeo
+   *
+   */
+
+  class Collapsible extends mixinBehaviors([], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          button {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin: 0;
+            padding: var(--nuxeo-collapsible-heading-padding, 4px);
+            background: transparent;
+            border: none;
+            cursor: pointer;
+            text-decoration: none;
+            font-size: 1rem;
+          }
+          /* to fix a blinking default style on safari */
+          button:active {
+            color: inherit;
+          }
+          .heading {
+            width: calc(100% - var(--nuxeo-collapsible-icon-width, 20px));
+            overflow-x: hidden;
+            text-align: start;
+          }
+          iron-icon {
+            --iron-icon-height: var(--nuxeo-collapsible-icon-height, 20px);
+            --iron-icon-width: var(--nuxeo-collapsible-icon-width, 20px);
+          }
+          .start {
+            flex-direction: row-reverse;
+            justify-content: flex-end;
+          }
+          .start-alignment {
+            margin-inline-start: var(--nuxeo-collapsible-icon-width, 20px);
+          }
+          iron-collapse {
+            padding: 0 var(--nuxeo-collapsible-heading-padding, 4px);
+          }
+        </style>
+
+        <button aria-expanded="[[expanded]]" on-tap="_toggle" class$="[[_changeChevronPosition(chevronAtStart)]]">
+          <slot class="heading" name="heading"></slot>
+          <iron-icon icon="[[_toggleIcon(expanded)]]" hidden$="[[!chevron]]"></iron-icon>
+        </button>
+        <iron-collapse
+          opened="[[expanded]]"
```

Comment:
we should use two way binding here and renamed ours to `opened` as well for consistency with other elements